src/parser/solidity.rs

```diff
@@ -1,4 +1,4 @@
-// auto-generated: "lalrpop 0.19.1"
+// auto-generated: "lalrpop 0.19.0"
```

Comment:
Your parser was generated using an older version of lalrpop. Please remove this change (you can remove `Cargo.lock` and rebuild).

Reply:
Haha, I ended up generating another file using a newer version of lalrpop. Is this ok?

Reply:
of course